README.md

```diff
@@ -1731,6 +1731,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sandra Duran](https://github.com/Erynoa)
   - [Sangamesh s](https://github.com/sangu143)
   - [Sangy K](https://github.com/sansk)
+  - [Sagnik Das](https://github.com/sagnik3788)
```

Comment:
Your name must be in line 1693, after Sagar Eknath Bangade.
Let us know if you need any help with updating. 🙂

Reply:
Should i make a new pr after changing this ?

Reply:
No, there's no need to create another PR. You can update the PR and the changes will be reflected here. 🙂